app/views/events/index.html.erb

```diff
@@ -1,24 +1,13 @@
-<h1>Events</h1>
+<div class="container">
+  <h1> <%= t('.events') %> </h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Duration</th>
-      <th>Started at</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @events.each do |event| %>
+  <table class= "table table-hover">
+    <thead>
       <tr>
-        <td><%= event.name %></td>
-        <td><%= event.duration %></td>
-        <td><%= event.started_at %></td>
-        <td><%= link_to 'Show', event %></td>
-        <td><%= link_to 'Edit', edit_event_path(event) %></td>
-        <td><%= link_to 'Destroy', event, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <th> <%= t('.name_event') %> </th>
+        <th><%= t('.duration') %> </th>
+        <th><%= t('.start_at') %> </th>
+        <th colspan="3"></th>
       </tr>
     <% end %>
```

Comment:
Is that working for you? You removed `<% @events.each do |event| %>` but I see that you still have `<% end %>`